$OpenBSD: patch-src_qt-console_bat_pro_in,v 1.1 2011/11/24 09:29:55 ajacoutot Exp $

Reorder LDFLAGS (prefer -L.libs over /usr/local/lib) so that bacula can
be rebuilt while already installed.

--- src/qt-console/bat.pro.in.orig	Thu Nov 24 08:43:23 2011
+++ src/qt-console/bat.pro.in	Thu Nov 24 08:44:08 2011
@@ -19,7 +19,7 @@ TEMPLATE     = app
 TARGET       = bat
 DEPENDPATH  += .
 INCLUDEPATH += .. . ./console ./restore ./select
-LIBS        += -L../lib -lbaccfg -lbac -L../findlib -lbacfind @OPENSSL_LIBS@
+QMAKE_LFLAGS+= -L../lib -lbaccfg -lbac -L../findlib -lbacfind @OPENSSL_LIBS@
 LIBTOOL_LINK = @QMAKE_LIBTOOL@ --silent --tag=CXX --mode=link
 LIBTOOL_INSTALL = @QMAKE_LIBTOOL@ --silent --mode=install
 QMAKE_LINK   = $${LIBTOOL_LINK} $(CXX)
